server/src/internalClusterTest/java/org/elasticsearch/search/fieldcaps/CCSFieldCapabilitiesIT.java

```diff
@@ -66,7 +66,7 @@ public void testFailuresFromRemote() {
             .setIndexFilter(new ExceptionOnRewriteQueryBuilder())
             .get();
         assertThat(response.getIndices()[0], equalTo(localIndex));
-        assertThat(response.getFailedIndices()[0], equalTo("remote_cluster:*"));
```

Comment:
We never used the actual string values for this thing in production, only the count so I made it count only.

Reply:
Does this change the REST response?

Reply:
No, the method is only used to get the array length in production, it's not used in serialisation.